plugins/inputs/systemd_units/README.md

```diff
@@ -0,0 +1,136 @@
+# Systemd Units Plugin
+
+The systemd_units plugin gathers systemd unit status on Linux. It relies on
+```systemctl list-units --all --type=service``` to collect data on service status.
+
+The results are tagged with the unit name and provide enumerated fields for
+loaded, active and running fields, indicating the unit health.
+
+This plugin is related to the [win_services module](../win_services/), which
+fulfills the same purpose on windows.
+
+In addition to services, this plugin can gather other unit types as well,
+see ```systemctl list-units --all --type help``` for possible options.
+
+### Configuration
+```
+[[inputs.systemd_units]]
+  ## The default timeout of 1s for systemctl execution can be overridden here:
+  # timeout = "1s"
+  # unittype = "service"
+```
+
+### Metrics
+- systemd_units:
+  - tags:
+    - name (string, unit name)
+  - fields:
+    - loaded (int, see below)
+    - active (int, see below)
+    - running (int, see below)
```

Comment:
Shouldn't this list be `load`, `active` and `sub`?

Reply:
you are right thanks for spotting this